### ISSUE TITLE
reproduce result

### ISSUE THREAD

**Issue by Iamal1** [ShuLiu1993/PANet#31]
I trained with batch size 8 on 8*GTX1080Ti.
`python tools/train_net_step.py --dataset coco2017 --cfg configs/panet/e2e_panet_R-50-FPN_2x_mask.yaml`
and only change IMAGE_PER_BATCH in config.py to 1 to get batch size 8.
At last I get box AP 0.3507 and mask AP 0.3148, which is much lower than the reported ones. I also downloaded the pretrained model and got the same result to this repo, so I think it's the problem of model.
But I'm confused about how to fix it. I use a smaller batch size but the lr is still 0.02 in config.yaml, should I make it smaller?(test it this weekend) And does anyone have other advice?
